config/initializers/assets.rb

```diff
@@ -18,4 +18,6 @@
   spice-html5-bower/spiceHTML5/spicearraybuffer.js
   webmks.css
   webmks.js
+  remote_consoles/unified_styles.css
+  remote_consoles/fullscreen_btn.js
```

Comment:
These lines make sure that the assets are reachable from `javascript_include_tag` and `stylesheet_link_tag`. You don't need the CSS as we're having this through `application.css`.